src/picknik_dual_ur_arm_config/launch/robot_drivers_to_persist.launch.py

```diff
@@ -0,0 +1,75 @@
+# Copyright 2023 PickNik Inc.
```

Comment:
I think you can probably reuse this file from one of the base configs so it's less copy pasta